rcldotnet/RCLdotnet.cs

```diff
@@ -198,9 +221,36 @@ public class RCLdotnet {
 
     public static Node CreateNode (string nodeName, string nodeNamespace) {
       IntPtr nodeHandle = IntPtr.Zero;
-      int ret = RCLdotnetDelegates.native_rcl_create_node_handle (ref nodeHandle, nodeName, nodeNamespace);
-      Node node = new Node (nodeHandle);
-      return node;
+      RMWRet ret = (RMWRet)RCLdotnetDelegates.native_rcl_create_node_handle (ref nodeHandle, nodeName, nodeNamespace);
+
+      switch (ret) {
+        case RMWRet.NodeNameValid:
+          Node node = new Node (nodeHandle);
+          return node;
+        case RMWRet.NodeNameInvalidIsEmptyString:
+          throw new NodeNameInvalidIsEmptyStringException(RMWGetErrorString());
```

Comment:
`rmw_reset_error` / `rcl_reset_error` needs to be called after calling `rmw_get_error_string` / `rcl_get_error_string`

Reply:
Ok, fixed. I have added these functions, and they are called after calling `*_get_error_string`